spec.bs

```diff
@@ -934,20 +934,26 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       1. Set |auctionConfig|'s [=auction config/seller signals=] to the result of
         [=serializing a JavaScript value to a JSON string=], given |result|.
     * To handle an error, set |auctionConfig|'s [=auction config/seller signals=] to failure.
-1. If |config|["{{AuctionAdConfig/directFromSellerSignals}}"] [=map/exists=]:
-  1. TODO: The receiving end of this isn't specified yet, so there is no place to put the computed value.
+1. If |config|["{{AuctionAdConfig/directFromSellerSignalsHeaderAdSlot}}"] [=map/exists=]:
   1. [=Handle an input promise in configuration=] given |auctionConfig| and
-    |config|["{{AuctionAdConfig/directFromSellerSignals}}"]:
+    |config|["{{AuctionAdConfig/directFromSellerSignalsHeaderAdSlot}}"]:
+    
+    Note: The JavaScript code calling {{Navigator/runAdAuction()}} is responsible for not resolving
+      |config|["{{AuctionAdConfig/directFromSellerSignalsHeaderAdSlot}}"] until a
+      {{WindowOrWorkerGlobalScope/fetch()}} call with the `{adAuctionHeaders: true}` option returns
+      a response, or alternatively, not calling {{Navigator/runAdAuction()}} until the
+      {{WindowOrWorkerGlobalScope/fetch()}} call returns a response.

```

Comment:
Posting a new comment, since I cannot unresolve the old one: I think TODO shoud should mention linking to the main section of my PR, not fetch() itself.

The old Github comment got resolved, but the TODO wasn't changed.

Reply:
I think I'll resolve the todo before landing this PR, so does not matter too much. I'll keep this open so that I remember what to link to.

Reply:
SGTM, just wanted to make sure it was tracked.

Reply:
Done in [this commit](https://github.com/WICG/turtledove/pull/774/commits/01d73c8d31e7e1c32ca54f2d068990cf26a829c3)